azure/scenarios/vm/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: vm
+runtime: go
+description: Agent VM
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
description: Agent Windows VM
```